### DISCUSSION TITLE
23.12.18 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#92] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 프로젝트 문서화
    - 발표 준비
- 내일은 어떤 일을 하나요?
    - 프로젝트 문서화

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 노래 조회 기능 구현
  - 아티스트 관련 api 명세서 작성
    - request body가 없는 경우 지울지 빈공간으로 둘지
  - 알림 통합 기능 수정 완료😎
- 내일은 어떤 일을 하나요?
  - 간단한 변수명, 메서드 수정
  - 프로젝트 문서화

**Reply by Jeonwoojae** [reply]
request body 지우도록 통일

**Reply by Jeonwoojae** [reply]
민주님 코드 리뷰도 하겠습니다!
